core/config/project_settings.h

```diff
@@ -113,7 +115,6 @@ class ProjectSettings : public Object {
 
 	Error _setup(const String &p_path, const String &p_main_pack, bool p_upwards = false);
 
-protected:
```

Comment:
Why? `_bind_methods()` is always `protected`.

Reply:
It's already in the `protected` block, scroll up a bit.